client-src/elements/chromedash-enterprise-release-notes-page.ts

```diff
@@ -298,7 +324,7 @@ export class ChromedashEnterpriseReleaseNotesPage extends LitElement {
         const queryParams = parseRawQuery(window.location.search);
         if (milestoneQueryParamKey in queryParams) {
           this.selectedMilestone = parseInt(
-            queryParams[milestoneQueryParamKey],
+            queryParams[milestoneQueryParamKey] as string,
```

Comment:
It would probably be better to fix the return type of `parseRawQuery` in `utils.js` rather than adding a cast here. I think it returns a `Record<string, string>`?